samples/Calculator/scenarios/scenario02/README.md

```diff
@@ -0,0 +1,45 @@
+**Scenario Description**
+
+Collect code coverage using cobertura report format without specifying configuration. Cobertura report format can be used to generate HTML report using [report generator](https://github.com/danielpalme/ReportGenerator). This format can be also used with [PublishCodeCoverageResults@2](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/publish-code-coverage-results-v2?view=azure-pipelines) in Azure DevOps pipelines.
+
+**Collect code coverage using command line**
+
+```shell
+git clone https://github.com/microsoft/codecoverage.git
+cd codecoverage/samples/Calculator/tests/Calculator.Core.Tests/
+dotnet test --collect "Code Coverage;Format=cobertura"
+```
+
+You can also use [run.ps1](run.ps1) to collect code coverage.
+
+**Collect code coverage inside github workflow**
+
+```shell
```

Comment:
Wonder if we could have 2 link here one pointing github yaml and another azdo and the 2 files in the same folder of the scenario. In that files we can in future add different "way" or configuration, like how to move files around at the end etc...